bin/tester.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+const colors = require('colors');
+const test = require('../lib/test');
+
+const path = '/Users/jerome/projects/testfiles/';
```

Comment:
You can use `process.cwd` to get the working directory.

Reply:
Thanks 👍